site/src/components/WorkspaceBuildProgress/WorkspaceBuildProgress.tsx

```diff
@@ -34,11 +34,13 @@ const estimateFinish = (
   p95: number,
 ): [number | undefined, string] => {
   const sinceStart = dayjs().diff(startedAt)
-  const secondsLeft = (est: number) =>
-    Math.max(
+  const secondsLeft = (est: number) => {
+    const max = Math.max(
       Math.ceil(dayjs.duration((1 - sinceStart / est) * est).asSeconds()),
       0,
     )
+    return isNaN(max) ? 0 : max
```

Comment:
Hmm, looking at the ticket, it seems like this will result in a string saying "Up to 0 seconds remaining" which seems a little off to me. What do you think about adjusting the text for this case so it says something like "Finishing up..." or similar instead?

Reply:
It actually hits the case that says "Anytime now..." I added a story called StartingZeroEstimate that should show that.

Reply:
Sweet!